flink-connectors/flink-connector-kafka/src/test/java/org/apache/flink/connector/kafka/source/KafkaSourceLegacyITCase.java

```diff
@@ -91,7 +91,7 @@ public void testMultipleSourcesOnePartition() throws Exception {
     // --- broker failure ---
 
     @Test
-    @Ignore("FLINK-28267")
+    @Disabled("FLINK-28267")
     public void testBrokerFailure() throws Exception {
```

Comment:
```suggestion
    void testBrokerFailure() throws Exception {
```
could be package private